addons/osio-addon/src/main/java/io/fabric8/launcher/osio/projectiles/OsioImportProjectileContext.java

```diff
@@ -23,7 +24,11 @@
 
     @FormParam("spacePath")
     @NotNull
-    @Pattern(message = "Space Path should start with a /", regexp = "\\/[a-zA-Z]*")
+    @Size(message = "Space Path must be in the range of 4 to 63 characters long", min = 5, max = 63)
```

Comment:
Text says 4-63 but code says 5-63

Reply:
I think it is considering the initial slash?

Reply:
Well then the code should say 5-64, shouldn't it?

Reply:
Ah good catch, yeah, I agree

Reply:
@ravsa what do you think? If you also agree, can you provide another PR with this fix? Thanks a million!

Reply:
nice catch, @quintesse 
yes, I will raise another PR to fix it.